apps/gitness/src/pages-v2/pull-request/pull-request-compare.tsx

```diff
@@ -167,6 +173,12 @@ export const CreatePullRequest = () => {
     queryParams: { page: 1, limit: 100, type: 'user', query: searchReviewers }
   })
 
+  const { labels: labelsList, values: labelsValues } = useGetRepoLabelAndValuesData({
+    query: searchLabel,
+    inherited: true,
```

Comment:
is just inherited?

Reply:
yes inherited includes project labels along with repo ones.